doc/source/examples/extended_examples/sfem/stochastic_fem.rst

```diff
@@ -0,0 +1,439 @@
+Numerous advantages / workflow possibilities that PyMAPDL affords users is demonstrated through this
+extended example. Important theoretical concepts are first explained before the example is presented.
+
+Introduction
+------------
+Often in a mechanical system, system parameters (geometry, materials, loads, etc.) and response parameters
+(displacement, strain, stress, etc) are taken to be deterministic. This simplification, while sufficient for a
+wide range of engineering applications, results in a crude approximation of actual system behaviour.
+
+To obtain a more accurate representation of a physical system, it is essential to consider the randomness
+in system parameters and loading conditions, along with the uncertainty in their estimation and their
+spatial variability. The finite element method is undoubtedly the most widely used tool for solving deterministic
+physical problems today and to account for randomness and uncertainty in the modeling of engineering systems,
+the stochastic finite element method (SFEM) was introduced.
+
+The stochastic finite element method (SFEM) extends the classical deterministic finite element approach
+to a stochastic framework, offering various techniques for calculating response variability. Among these,
+the Monte Carlo simulation (MCS) stands out as the most prominent method. Renowned for its versatility and
+ease of implementation, MCS can be applied to virtually any type of problem in stochastic analysis.
+
+Random variables vs stochastic processes
+----------------------------------------
+A distinction between random variables and stochastic processes is attempted in this section. Explaining these
+concepts is important since they are used for modelling the system randomness. Random variables are easier to
+understand from elementary probability theory, the same cannot be said for stochastic processes. Readers are
+advised to consult books on SFEM if the explanation here seems too brief. [1]_ and [2]_ are recommended resources.
+
+Random variables
+~~~~~~~~~~~~~~~~
+**Definition:** A random variable is a rule for assigning to every possible outcome :math:`\theta` of an experiment a
+number :math:`X(\theta)`. For notational convenience, the dependence on :math:`\theta` is usually dropped and the
+random variable is written as :math:`X`.
+
+Practical example
++++++++++++++++++
+Imagine a beam with a concentrated load :math:`P` applied at a specific point. The value of :math:`P`
+is uncertain—it could vary due to manufacturing tolerances, loading conditions, or measurement errors. Mathematically,
+:math:`P` is a random variable:
+
+.. math:: P : \Theta \longrightarrow \mathbb{R}
+
+where :math:`\Theta` is the sample space of all possible loading scenarios, and :math:`\mathbb{R}` represents the set of
+possible load magnitudes. For example, :math:`P` could be modeled as a random variable with a probability density
```

Comment:
```suggestion
In the preceding equation, :math:`\Theta` is the sample space of all possible loading scenarios, and :math:`\mathbb{R}` represents the set of
possible load magnitudes. For example, :math:`P` could be modeled as a random variable with a probability density
```
While the construction you use is common in white papers, it is difficult to read online. It helps if the content is written as full sentences rather than fragments.